assets/xml/objects/object_tab.xml

```diff
@@ -1,63 +1,63 @@
-        <DList Name="object_tab_DL_005988" Offset="0x5988" />
-        <DList Name="object_tab_DL_005A78" Offset="0x5A78" />
-        <DList Name="object_tab_DL_005B60" Offset="0x5B60" />
-        <Texture Name="object_tab_TLUT_005CA8" OutName="tlut_005CA8" Format="rgba16" Width="16" Height="16" Offset="0x5CA8" />
-        <Texture Name="object_tab_Tex_005EA8" OutName="tex_005EA8" Format="rgba16" Width="16" Height="16" Offset="0x5EA8" />
-        <Texture Name="object_tab_Tex_0060A8" OutName="tex_0060A8" Format="ci8" Width="8" Height="16" Offset="0x60A8" />
-        <Texture Name="object_tab_Tex_006128" OutName="tex_006128" Format="ci8" Width="16" Height="32" Offset="0x6128" />
-        <Texture Name="object_tab_Tex_006328" OutName="tex_006328" Format="ci8" Width="8" Height="8" Offset="0x6328" />
-        <Texture Name="object_tab_Tex_006368" OutName="tex_006368" Format="ci8" Width="8" Height="16" Offset="0x6368" />
-        <Texture Name="object_tab_Tex_0063E8" OutName="tex_0063E8" Format="ci8" Width="8" Height="8" Offset="0x63E8" />
-        <Texture Name="object_tab_Tex_006428" OutName="tex_006428" Format="ci8" Width="32" Height="32" Offset="0x6428" />
-        <Texture Name="object_tab_Tex_006828" OutName="tex_006828" Format="ci8" Width="16" Height="16" Offset="0x6828" />
-        <Texture Name="object_tab_Tex_006928" OutName="tex_006928" Format="ci8" Width="32" Height="32" Offset="0x6928" />
-        <Texture Name="object_tab_Tex_006D28" OutName="tex_006D28" Format="ci8" Width="32" Height="32" Offset="0x6D28" />
-        <Texture Name="object_tab_Tex_007128" OutName="tex_007128" Format="ci8" Width="16" Height="16" Offset="0x7128" />
-        <Texture Name="object_tab_Tex_007228" OutName="tex_007228" Format="ci8" Width="16" Height="16" Offset="0x7228" />
-        <Texture Name="object_tab_Tex_007328" OutName="tex_007328" Format="i8" Width="32" Height="32" Offset="0x7328" />
-        <Texture Name="object_tab_Tex_007728" OutName="tex_007728" Format="rgba16" Width="8" Height="16" Offset="0x7728" />
-        <Texture Name="object_tab_Tex_007828" OutName="tex_007828" Format="ci8" Width="8" Height="8" Offset="0x7828" />
-        <Texture Name="object_tab_Tex_007868" OutName="tex_007868" Format="ci8" Width="8" Height="16" Offset="0x7868" />
-        <Texture Name="object_tab_Tex_0078E8" OutName="tex_0078E8" Format="i8" Width="16" Height="16" Offset="0x78E8" />
-        <Texture Name="object_tab_Tex_0079E8" OutName="tex_0079E8" Format="i8" Width="8" Height="8" Offset="0x79E8" />
-        <Texture Name="object_tab_Tex_007A28" OutName="tex_007A28" Format="rgba16" Width="16" Height="16" Offset="0x7A28" />
+        <Animation Name="gBartenIdleAnim" Offset="0x758" />
+        <DList Name="gBartenPelvisDL" Offset="0x3E10" />
+        <DList Name="gBartenLeftHandDL" Offset="0x3EF0" />
+        <DList Name="gBartenLeftBroomDL" Offset="0x4270" />
+        <DList Name="gBartenLeftLowerArmDL" Offset="0x4438" />
```

Comment:
For humanoid characters, I tend to use `Forearm` instead of `LowerArm`. This applies to `gBartenRightLowerArmDL` and the associated limbs too.